src/theme/components/AccordionTheme.ts

```diff
@@ -0,0 +1,13 @@
+import { Components } from "@mui/material"
+
+export default function AccordionTheme(): Components {
+  return {
+    MuiAccordion: {
+      styleOverrides: {
+        root: {
+          borderRadius: "8px !important",
```

Comment:
does this styling not work without `!important`?

Reply:
On `Addordion.js` of Mui library, the borderRadius was defined like the following.

```
  borderRadius: 0,
  '&:first-of-type': {
    borderTopLeftRadius: theme.shape.borderRadius,
    borderTopRightRadius: theme.shape.borderRadius
  },
  '&:last-of-type': {
    borderBottomLeftRadius: theme.shape.borderRadius,
    borderBottomRightRadius: theme.shape.borderRadius,
    // Fix a rendering issue on Edge
    '@supports (-ms-ime-align: auto)': {
      borderBottomLeftRadius: 0,
      borderBottomRightRadius: 0
    }
  }
```

So, to override this CSS, we need to give on the '&:first-of-type' and '&:last-of-type'.
To reduce the code, I gave !important.
Also, Mui support to customize of `styled` like styled-component library, it is not good to use `!important`.

So, I give borderRadius value of shape of theme, and remove the !important.